README.md

```diff
@@ -21,11 +21,12 @@ The frontend transforms a model specification into M2-ISA-R's internal architect
 ## Usage
 M2-ISA-R v2 is divided into two separate tools: Parser and Writer. These are described below, TL;DR version:
 
-- To parse a CoreDSL description: `python -m m2isar.frontends.coredsl.parser [-j threads] path/to/input`
-- To generate ETISS Architecture: `python -m m2isar.backends.etiss.writer -s path/to/input`
+- To parse a CoreDSL description: `python -m m2isar.frontends.coredsl.parser [-j threads] path/to/input/<top_level>.core_desc`
+- To generate ETISS Architecture: `python -m m2isar.backends.etiss.writer -s path/to/input/gen_model/<top_level>.m2isarmodel`
 
 Notes:
-- `path/to/input` stays the same for both calls if the same model is compiled and should point to the top-level CoreDSL file
+- `path/to/input/<top_level>.core_desc` should point to the top-level CoreDSL file (i.e. the file where `Core`s are defined), where `<top_level>` is the name of the top-level file
+- `path/to/input/gen_model/<top_level>.m2isarmodel` should point to the internal architecture model file generated by the frontend
 - The intermediate and output files are put into `path/to/input/gen_model` and `path/to/input/gen_output`, respectively
```

Comment:
Probably a note about the generated output products would be helpful as well.

Reply:
Yes exactly. Explain where `coredsl_parser` puts its metamodel, and where `etiss_writer` puts its generated code.

Reply:
done with f388b96